app/src/components/SignalStrengthLegend.js

```diff
@@ -15,35 +15,50 @@ import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
 import ListItemAvatar from '@material-ui/core/ListItemAvatar';
 import ListItemText from '@material-ui/core/ListItemText';
+import {DEFAULT_RANGE, TURBO_HEATMAP, getRGBCustom} from '../utils/ColorUtils';
 
-type Bucket = {
-  color: number[],
-  maxValue: number,
-};
+import type {HeatmapType} from '../utils/ColorUtils';
 
 type Props = {
-  buckets: Array<Bucket>,
+  range: [number, number],
+  heatmap: HeatmapType,
```

Comment:
The alternative to these props was passing in:
- getColor()
- range (min, max)
- step count